src/PharoThings-Hardware-Core.package/PotBoard.class/instance/extendConnector.with..st

```diff
@@ -0,0 +1,8 @@
+initialization
+extendConnector: aString with: pinsArray
+	| connector pinsArrayExisting |
+	pinsArrayExisting  := (self connectorNamed: aString) pins.
+	connector := PotBoardConnector named: aString in: self with: pinsArrayExisting , pinsArray.
+	connectors removeFirst.
```

Comment:
it is still buggy because existing connector is not necessary the first one

Reply:
I removed this and created a method #addPins: named: to retrieve the old pins and concatenated them with the new one